cmd/livepeer/livepeer.go

```diff
@@ -633,10 +634,19 @@ func main() {
 				timeWatcher,
 				cfg,
 			)
+			n.SetTicketEV(ev)
```

Comment:
Isn't that line setting the fixed face value? I think we wanted to allow only setting max face value, why do we need this line then?